app/assets/javascripts/chf_ga_events.js

```diff
@@ -0,0 +1,14 @@
+// Sufia/hyrax JS for sending event tracking to Google Analytics is a bit inflexible.
+// https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
```

Comment:
The benefit is if some other template from one of the upstream gems, now or in the future, assumes the upstream GA code is there, and we don't notice it's relying on it. I'm not sure, I see both sides. I prefer not overriding upstream code when I don't have to, as a general principle.